netutils/config/parser.py

```diff
@@ -603,3 +603,78 @@ class JunosConfigParser(BaseSpaceConfigParser):
 
     comment_chars = []
     banner_start = []
+
+
+class ASAConfigParser(CiscoConfigParser, BaseSpaceConfigParser):
```

Comment:
CiscoConfigParser already inherits from BaseSpaceConfigParser, e.g. `class CiscoConfigParser(BaseSpaceConfigParser):`

This should suffice `class ASAConfigParser(CiscoConfigParser):`

Reply:
This is due to using IOSConfigParser as an example and it inherits from those two classes. I will correct it.